README.md

```diff
@@ -62,6 +62,10 @@ $pdf->send();
 
 // ... or send to client as file download
 $pdf->send('test.pdf');
+
+// ... or you can get the raw pdf as a string
+if (($pdfText = (string) $pdf) === false) echo 'Could not create PDF Test Page. '.$pdf->getError();
```

Comment:
Could you please add a line break + indent before `echo ...` here? You can then simply force push an amended commit, if you like.